.drone.yml

```
@@ -902,8 +902,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     pull: always
     settings:
-      auto_tag: true
-      auto_tag_suffix: linux-amd64
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
+      auto_tag: false
+      tags: ${DRONE_TAG}-linux-amd64
```

```suggestion
      tags: ${DRONE_TAG##v}-linux-amd64
```
[I think ](https://gitea.com/gitea/drone-docker/src/branch/master/tags.go#L61-L71) get's trimed on auto_tag - we have to imitate it